lib/rules/indentation/html.test.js

```diff
@@ -631,3 +632,277 @@ a {
 		},
 	],
 })
+
+testRule({
+	plugins,
+	ruleName,
+	config: [
+		2,
+		{
+			baseIndentLevel: 1,
+		},
+	],
+	customSyntax: `postcss-html`,
+	fix: true,
+
+	accept: [
+		{
+			code: stripIndent(`
+				<html>
+				  <head>
+				    <style>
+				      a {
+				        display: block;
+				      }
+				    </style>
+				  </head>
+				</html>
+			`),
+		},
+		{
+			code: stripIndent(`
+				<div>
+				  <style>
+				    a {
+				      color: pink;
+				    }
+				  </style>
+				</div>
+			`),
+		},
+	],
+
+	reject: [
+		{
+			code: stripIndent(`
+				<html>
+				  <head>
+				    <style>
+				    a {
+					display: block;
+				      }
+				    </style>
+				  </head>
+				</html>
+			`),
+			fixed: stripIndent(`
+				<html>
+				  <head>
+				    <style>
+				      a {
+				        display: block;
+				      }
+				    </style>
+				  </head>
+				</html>
+			`),
+			warnings: [
+				{
+					message: messages.expected(`6 spaces`),
+					line: 4,
+					column: 5,
+				},
+				{
+					message: messages.expected(`8 spaces`),
+					line: 5,
+					column: 2,
+				},
+			],
+		},
+		{
+			code: stripIndent(`
+				<div>
+				  <style>
+				a {
+				  color: pink;
+					}
+				  </style>
+				</div>
+			`),
+			fixed: stripIndent(`
+				<div>
+				  <style>
+				    a {
+				      color: pink;
+				    }
+				  </style>
+				</div>
+			`),
+			warnings: [
+				{
+					message: messages.expected(`4 spaces`),
+					line: 3,
+					column: 1,
+				},
+				{
+					message: messages.expected(`4 spaces`),
+					line: 5,
+					column: 2,
+				},
+				{
+					message: messages.expected(`6 spaces`),
+					line: 4,
+					column: 3,
+				},
+			],
+		},
+	],
+})
+
+testRule({
+	plugins,
+	ruleName,
+	config: [
+		`tab`,
+		{
+			baseIndentLevel: 1,
+		},
+	],
+	customSyntax: `postcss-html`,
+	fix: true,
+
+	accept: [
+		{
+			code: stripIndent(`
+				<html>
+				\t<head>
```

Comment:
Is there any reason why you put the tab here as `\t` rather than just a symbol?
If not, I'd prefer to see tabs as straight characters - it's easier to scan the code that way.
As I can see, everything works the same if you do that.